internal/pkg/service/appsproxy/http/router_test.go

```diff
@@ -534,7 +535,7 @@ func TestAppProxyRouter(t *testing.T) {
 				require.NoError(t, err)
 				response, err = client.Do(request)
 				require.NoError(t, err)
-				require.Equal(t, http.StatusFound, response.StatusCode)
+				require.Equal(t, http.StatusForbidden, response.StatusCode)
 			},
```

Comment:
I spent a lot of time on this, and I don't know.

Is it possible that there was a bug, and I fixed it unplanned? Jachym?

Some logs:

```
{"level":"info","time":"2024-04-15T06:47:31.492Z","message":"[2024/04/15 06:47:31] [oauthproxy.go:185] Cookie settings: name:_oauth2_proxy secure(https):true httponly:true expiry:168h0m0s domains:oidc.hub.keboola.local path:/ samesite:lax refresh:disabled\n"}
{"level":"info","time":"2024-04-15T06:47:31.493Z","message":"[2024/04/15 06:47:31] [oauthproxy.go:1029] No valid authentication in request. Initiating login.\n"}
{"level":"info","time":"2024-04-15T06:47:31.494Z","message":"127.0.0.1:56630 - 31916724-e772-46a6-b916-6434f75249a8 - - [2024/04/15 06:47:31] oidc.hub.keboola.local GET - \"/\" HTTP/2.0 \"Go-http-client/2.0\" 302 446 0.001\n"}
{"level":"info","time":"2024-04-15T06:47:31.496Z","message":"[2024/04/15 06:47:31] [provider.go:55] Performing OIDC Discovery...\n"}
{"level":"warn","time":"2024-04-15T06:47:31.496Z","message":"unable to create oauth provider for app \"oidc\" \"\": could not create provider data: error building OIDC ProviderVerifier: could not get verifier builder: error while discovery OIDC configuration: failed to discover OIDC configuration: error performing request: Get \"http://127.0.0.1:46815/oidc/.well-known/openid-configuration\": dial tcp 127.0.0.1:46815: connect: connection refused","exceptionId":"proxy-Z931jE3HReqllCY","proxy.appid":"oidc"}
{"level":"warn","time":"2024-04-15T06:47:31.497Z","message":"application \"oidc\" has misconfigured OAuth2 provider","exceptionId":"proxy-Z931jE3HReqllCY","proxy.appid":"oidc"}
{"level":"info","time":"2024-04-15T06:47:31.497Z","message":"exiting (test cleanup)"}
{"level":"info","time":"2024-04-15T06:47:31.497Z","message":"exited"}

```

Reply:
Yes please.

Reply:
Thanks, I will look.

Reply:
Thanks, Fixed :heavy_check_mark: https://github.com/keboola/keboola-as-code/pull/1701/commits/d202a23230680538048fa3266dd815ba5adbc3a1